lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -353,6 +356,24 @@ spec = do
             [ expectErrorMessage errMsg400MalformedTxPayload
             , expectResponseCode HTTP.status400
             ]
+
+    it "BYRON_MIGRATE_07 - migrate to inaproppriate addresses" $ \ctx -> do
+        let addrsInvalid :: [Text] =
+                [ "DdzFFzCqrhtCNjPk5Lei7E1FxnoqMoAYtJ8VjAWbFmDb614nNBWBwv3kt6QHJa59cGezzf6piMWsbK7sWRB5sv325QqWdRuusMqqLdMt"
+                , "37btjrVyb4KFbHcx5z2vjmdDBm6K68A5wTDJaL9XB8E8wcVfB6o8bJazKLC9tfrNTN3B4uCppR3MS8BYGaz92AqQtUVAKzGBUSQSooFW5pd84jc6Jj"
+                ]
+        forM_ addrsInvalid $ \addr -> do
+            sWallet <- fixtureRandomWallet ctx
+            r <- request @[ApiTransaction n] ctx
+                (Link.migrateWallet sWallet)
+                Default
+                (Json [json|
+                    { passphrase: #{fixturePassphrase}
+                    , addresses: [#{addr}]
+                    }|])
+            verify r
+                [ expectResponseCode @IO HTTP.status500
```

Comment:
Could we expect `HTTP.status400` and set the test to pending?

Reply:
Ok, corrected.